qemu/tests/drive_mirror.py

```diff
@@ -134,25 +134,11 @@ def reopen(self):
         params = self.parser_test_args()
         target_format = params["image_format"]
         timeout = params["reopen_timeout"]
-
-        def is_opened():
-            try:
-                device = self.vm.get_block({"file": self.target_image})
-                ret = (device == self.device)
-                if self.vm.monitor.protocol == "qmp":
-                    ret &= bool(self.vm.monitor.get_event("BLOCK_JOB_COMPLETED"))
-                return ret
-            except Exception:
-                return False
-
         error.context("reopen new target image", logging.info)
         if self.vm.monitor.protocol == "qmp":
             self.vm.monitor.clear_event("BLOCK_JOB_COMPLETED")
         self.vm.block_reopen(self.device, self.target_image, target_format)
-        opened = utils_misc.wait_for(is_opened, first=3.0, timeout=timeout)
-        if not opened:
-            msg = "Target image not used,wait timeout in %ss" % timeout
-            raise error.TestFail(msg)
```

Comment:
@vivianQizhu  after this change,  it will caused regression that original test will failed on rhel6 host.